### PR TITLE
Updated release versions to 4.45 for azure-cosmos and azure-cosmos-encryption

### PR DESCRIPTION
**Pull request by kushagraThapar**
Updated release versions to 4.45 for azure-cosmos and azure-cosmos-encryption

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosAsyncClientTest.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 public abstract class CosmosAsyncClientTest implements ITest {
 
     public static final String ROUTING_GATEWAY_EMULATOR_PORT = ":8081";
-    public static final String COMPUTE_GATEWAY_EMULATOR_PORT = ":9999";
+    public static final String COMPUTE_GATEWAY_EMULATOR_PORT = ":8081";
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosAsyncClientTest.java:17]: That will result in other test failures

File: sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,8 @@
...
 
-### 2.1.0-beta.1 (Unreleased)
-
-#### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
-
+### 2.1.0 (2023-05-12)
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md:3]: NPE fix changelog is missing

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Release History
 
-### 4.45.0-beta.1 (Unreleased)
+### 4.45.0 (2023-05-12)
 
 #### Features Added
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:5]: UUID fix is missing

Comment (jeet1995) [reply sdk/cosmos/azure-cosmos/CHANGELOG.md:5]: I suppose it is included under the "Other Changes" heading - **Removed synchronized locking on generating random UUIDs**

Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/CHANGELOG.md:5]: its in other changes section.

File: sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md (hunk 2 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md:3]: thanks, added.

File: sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosAsyncClientTest.java (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosAsyncClientTest.java]: reverted back, thanks!

File: sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/EndToEndTimeOutValidationTests.java [hunk 4]
```diff
@@ -248,7 +248,7 @@ public void clientLevelEndToEndTimeoutPolicyInOptionsShouldTimeout() {
             // with injected delay
             CosmosQueryRequestOptions queryRequestOptions = new CosmosQueryRequestOptions()
                 .setCosmosE2EOperationRetryPolicyConfig(
```
Comment (jeet1995) [inline sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/EndToEndTimeOutValidationTests.java:250]: `setCosmosE2EOperationRetryPolicyConfig` to `setCosmosEndToEndOperationRetryPolicyConfig`

Comment (jeet1995) [inline sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/EndToEndTimeOutValidationTests.java:250]: `setCosmosE2EOperationRetryPolicyConfig` to `setCosmosEndToEndOperationRetryPolicyConfig`

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

**Review by jeet1995 — APPROVED** [review]
LGTM

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM - Thanks!

**Pull request merged**